rules/windows/registry/registry_set/registry_set_persistence_shim_database_susp_application.yml

```diff
@@ -0,0 +1,32 @@
+title: Suspicious Shim Database Patching Activity
+id: bf344fea-d947-4ef4-9192-34d008315d3a
+status: experimental
+description: Detects installation of new shim databases that try to patch section of know processes for potential process injection or persistence.
```

Comment:
```suggestion
description: Detects installation of new shim databases that try to patch a section of know processes for potential process injection or persistence.
```
or section**s**?

Reply:
Yeah could be sections. Updated the suggestion